lms/views/predicates/_lti_launch.py

```diff
@@ -42,7 +44,40 @@ def __call__(self, context, request):
         return has_document_url == self.value
 
 
-class BlackboardCopied(Base):
+class _CourseCopied(Base, ABC):
```

Comment:
Added a base class for `BlackboardCopied` and `BrightspaceCopied`, since they're the same except for the param name.

I'm not sure that I think we should actually be using `ABC`'s, we're just writing an app not a framework, and I think we can just do "informal" protocols and abstract base classes. Perhaps with `raise NotImplementedError()`. But the pre-existing `Base` class is already an `ABC` and that forces us to make this new abstract class an `ABC` too otherwise PyLint complains. For the sake of keeping this PR small I just went along with it. Refactoring `Base` to not be an `ABC` could be done separately if we want to